src/sagemaker/tensorflow/deploying_tensorflow_serving.rst

```diff
@@ -66,6 +64,17 @@ If you already have existing model artifacts in S3, you can skip training and de
 
   predictor = model.deploy(initial_instance_count=1, instance_type='ml.c5.xlarge')
 
+You can also deploy the model with Amazon Elastic Inference, which allows you to attach low-cost GPU accelerators to your Amazon EC2 and SageMaker instances.
```

Comment:
We should be consistent with where the EI code block is introduced in the documentation.

Either the current docs or this should change: https://github.com/aws/sagemaker-python-sdk/blob/aa7335c6556d122173030e5cf67b46d1b4b46b3c/src/sagemaker/tensorflow/deploying_python.rst#deploying-from-an-estimator